catalogue/webapp/pages/concept.tsx

```diff
@@ -115,7 +117,7 @@ export const ConceptPage: NextPage<Props> = ({
     // TODO fill meta information; wait for confirmation on description
     <CataloguePageLayout
       title={conceptResponse.label}
-      description={`Find out more about ${conceptResponse.label} by browsing related works and images from Wellcome Collection.`}
+      description={`Find books, manuscripts, paintings, illustrations, photos and unpublished archives about ${conceptResponse.label}, many of them with free online access.`}
```

Comment:
Can we put this in our microcopy file?

Reply:
I believe the microcopy file is for generic descriptions of landing pages, whereas this one is for each specific concept, passing a variable in. So from what I understand, for example `/books` passes in `pageDescriptions.books`, but `books/[id]` passes in a dynamic value (`book.metadataDescription || book.promo?.caption || ''`), granted, based on Prismic content, but the logic would still stand?

Actually I'm thinking maybe we'll replace it with the concept's description once we get that through?

Reply:
> I believe the microcopy file is for generic, static descriptions of landing pages

I think of it as a place for all the bits of text on the site that don't come from an external system, so they can be reviewed (and edited) together. Right now it's mostly static descriptions, but my aspiration was that we'd move all the bits of copy like this there.

Maybe this would have to be a function rather than a static string, but I think `function conceptDescription(label: string): string` would still work in that file.

Reply:
As long as we don't think it goes against

> This file is simple enough that a non-developer with a GitHub account might be able to edit it themselves, using the GitHub web interface.

I'm happy to change it, but yeah it will need to be a separate function. I'd be tempted to make it more generic for future use, although that feels like overengineering it for now 🤔

Reply:
I've changed it but also moved in `works`' and `images`', hope that's alright, noticed they were static too!